src/test/java/com/skynest/LoginTest.java

```diff
@@ -9,60 +9,60 @@
 
 public class LoginTest extends BaseTest {
 
+    @DataProvider(name = "invalidCredentialCombinations")
+    public static Object[][] invalidCredentials() {
+        return new Object[][]{
+                {"invalidemail@yahoo.com", "System123"},
+                {"invalidemail@yahoo.com", "invalidpass"},
+                {"invalidemail@yahoo.com", ""},
+                {"", "System123"},
+                {"", "invalidpass"},
+                {"", ""}
+        };
+    }
+
     @Test
     void verified_user_should_be_successfully_logged_in() {
         LoginRequest body = new LoginRequest("yagaj78380@jrvps.com", "System123");
-        Response response = skyNestBackendClient.login(body);
-        response.then().statusCode(SC_OK);
+        Response loginResponse = skyNestBackendClient.login(body);
+        loginResponse.then().statusCode(SC_OK);
     }
 
     @Test
     void not_verified_user_should_not_be_logged_in() {
         LoginRequest body = new LoginRequest("miwix46540@jrvps.com", "System123");
-        Response response = skyNestBackendClient.login(body);
-        response.then().statusCode(SC_FORBIDDEN);
+        Response loginResponse = skyNestBackendClient.login(body);
+        loginResponse.then().statusCode(SC_FORBIDDEN);
     }
 
     @Test
     void user_should_not_be_logged_in_entering_valid_email_and_invalid_password() {
         LoginRequest body = new LoginRequest("yagaj78380@jrvps.com", "invalidpass");
-        Response response = skyNestBackendClient.login(body);
-        response.then().statusCode(SC_UNAUTHORIZED);
+        Response loginResponse = skyNestBackendClient.login(body);
+        loginResponse.then().statusCode(SC_UNAUTHORIZED);
     }
 
     @Test
     void user_should_not_be_logged_in_entering_valid_email_and_blank_password() {
         LoginRequest body = new LoginRequest("yagaj78380@jrvps.com", "");
-        Response response = skyNestBackendClient.login(body);
-        response.then().statusCode(SC_UNAUTHORIZED);
+        Response loginResponse = skyNestBackendClient.login(body);
+        loginResponse.then().statusCode(SC_UNAUTHORIZED);
     }
 
     @Test(dataProvider = "invalidCredentialCombinations")
     void user_should_not_log_in_with_invalid_credentials(String email, String password) {
         LoginRequest body = new LoginRequest(email, password);
-        Response response = skyNestBackendClient.login(body);
-        response.then().statusCode(SC_NOT_FOUND);
+        Response loginResponse = skyNestBackendClient.login(body);
+        loginResponse.then().statusCode(SC_NOT_FOUND);
     }
 
     @Test
     void user_should_be_locked_after_five_invalid_attempts() {
```

Comment:
Might be preferable to have the number of attempts be an argument for this test method. This way we could get rid of having to change the name of the test along the implementation should the number change. Perhaps something like: user_should_be_locked_after_number_of_invalid_attempts(int maxAttempts) { ... }, and have the number passed via @DataProvider. This has the added benefit of having the actual number used appear in the test report itself